app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlController.kt

```diff
@@ -343,8 +343,22 @@ class DefaultSessionControlController(
 
     override fun handleSelectTopSite(url: String) {
         invokePendingDeleteJobs()
+        when (url) {
+            SupportUtils.ADD_SITE_MOCK_URL -> {
+                val directions =
+                    HomeFragmentDirections.actionHomeFragmentToAddTopSiteFragment()
+                navController.navigate(directions)
+                return
+            }
+            SupportUtils.POCKET_TRENDING_URL -> {
+                metrics.track(Event.PocketTopSiteClicked)
+            }
+        }
+
         metrics.track(Event.TopSiteOpenInNewTab)
-        if (url == SupportUtils.POCKET_TRENDING_URL) { metrics.track(Event.PocketTopSiteClicked) }
+        if (url == SupportUtils.POCKET_TRENDING_URL) {
```

Comment:
We could move this to the above when (url) mock to neatly handle all these url cases.

Reply:
@gabrielluong Thanks for noticing, I have already moved this in the when, just forgot to remove the original tracking.